ci/build.sh

```diff
@@ -192,13 +191,14 @@ function generate_overrides() {
   echo "Generating setup job yaml from helm chart..."
   echo "# This file is auto-generated." > deploy/kubernetes/setup-sumologic.yaml.tmpl
 
-  with_files=$(find deploy/helm/sumologic/templates/setup/ -maxdepth 1 -iname "*.yaml" | sed 's#deploy/helm/sumologic/templates#-x templates#g' | sed 's/yaml/yaml \\/g')
+  # Get list of templates available for default values.yaml
+  with_files=$(find deploy/helm/sumologic/templates/setup -maxdepth 1 -iname "*.yaml" ! -path "*/setup-scc.yaml" | sed 's#deploy/helm/sumologic/templates#-s templates#g' | sed 's/yaml/yaml /g')
```

Comment:
`! -path "*/setup-scc.yaml"`?

Reply:
do not take `*/setup-scc.yaml` during generating the `deploy/kubernetes/sumologic.yaml.tmpl`
It is disabled by default in the `values.yaml` so helm3 doesn't see it as template